ozaria/site/components/play/PageIntroLevel/index.vue

```diff
@@ -0,0 +1,134 @@
+<script>
+import api from 'core/api'
+import interactivesComponent from '../../interactive/PageInteractive'
+import cinematicsComponent from '../../cinematic/PageCinematic'
+import { defaultCodeLanguage } from 'ozaria/site/common/ozariaUtils'
+
+module.exports = Vue.extend({
+  props: {
+    introLevelIdOrSlug: {
+      type: String,
+      required: true,
+      default: ''
+    },
+    courseInstanceId: {
+      type: String
+    },
+    codeLanguage: {
+      type: String
+    },
+    courseId: {
+      type: String
+    },
+  },
+  data: () => ({
+    introLevelData: {},
+    introLevelSession: {},
+    introContent: [],
+    currentContent: {},
+    currentIndex: 0,
+    language: ''
+  }),
+  components: {
+    'interactives-component': interactivesComponent,
+    'cinematics-component': cinematicsComponent
+    // TODO add when ready
+    // 'cutscene-video-component': cutsceneVideoComponent,
+    // 'avatar-selection-screen': avatarSelectionScreen
+  },
+  async created () {
+    if (!me.hasIntroLevelAccess()) {
+      alert('You must be logged in as an admin to use this page.')
+      return application.router.navigate('/', { trigger: true })
+    }
+    try {
+      this.introLevelData = await api.levels.getByIdOrSlug(this.introLevelIdOrSlug)
+      if (me.isSessionless()) {
```

Comment:
Same comment as above re this call.  Can you add a quick comment on why we need this here?